src/lib/gadgets/gadgets.ts

```diff
@@ -62,4 +62,58 @@ const Gadgets = {
   rot(field: Field, bits: number, direction: 'left' | 'right' = 'left') {
     return rot(field, bits, direction);
   },
+
+  /**
+   * Performs a left shift operation on the provided {@link Field} element.
+   * This is akin to the `<<` shift operation in JavaScript, where bits are moved to the left.
```

Comment:
In light of https://github.com/o1-labs/o1js-bindings/pull/191/files#r1370549886 we should probably slightly change this part

Reply:
Addressed in https://github.com/o1-labs/o1js/pull/1194/commits/575920dbd2a4e50564ca422de7e7da4cc9b57542, how does it sound now? :D

Reply:
sounds good!